apps/speed.c

```diff
@@ -3606,6 +3883,27 @@ int speed_main(int argc, char **argv)
                    dsa_results[k][0], dsa_results[k][1]);
     }
 #endif
+#ifndef OPENSSL_NO_DH
+    testnum = 1;
+    for (k = 0; k < FFDH_NUM; k++) {
+        if (!ffdh_doit[k])
+            continue;
+        if (testnum && !mr) {
+            printf("%23sop     op/s\n", " ");
+            testnum = 0;
+        }
+        if (mr)
+            printf("+F5:%u:%u:%f:%f\n",
```

Comment:
Again you should move the whole added hunk to be the last and change the F5 to be F8. And you need to add appropriate else if block with F8 in do_multi().

Reply:
and done, also verified that -multi works as expected